docs/conf.py

```diff
@@ -50,7 +83,7 @@
 ### Configuration for extensions
 
 # Used for related links
-if not 'discourse_prefix' in html_context and 'discourse' in html_context:
+if 'discourse_prefix' not in html_context and 'discourse' in html_context:
```

Comment:
Fix some lint issues, already added enough exceptions in `pyproject.toml`; same below.